src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -667,6 +667,7 @@ export default class Pilot {
             this._mcp.setHeadingFieldValue(runway.angle);
         }
 
+        // FIXME: should this be LNAV or HOLD?
         if (this._mcp.headingMode === MCP_MODE.HEADING.OFF) {
             this._mcp.setHeadingLnav();
         }
```

Comment:
LNAV. They won't take off without being cleared on their route, and if they haven't received any specific heading instructions, the expectation should be that they do whatever is on their flightplan. So LNAV is :+1: